src/modules/squeeze/squeeze.service.ts

```diff
@@ -0,0 +1,36 @@
+import { BadRequestException, Injectable, UnprocessableEntityException } from '@nestjs/common';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Squeeze } from './entities/squeeze.entity';
+import { SqueezeRequestDto } from './dto/squeeze.dto';
+import { CreateSqueezeMapper } from './mapper/create-squeeze.mapper';
+import { SqueezeMapper } from './mapper/squeeze.mapper';
+
+@Injectable()
+export class SqueezeService {
+  constructor(
+    @InjectRepository(Squeeze)
+    private readonly SqueezeRepository: Repository<Squeeze>
+  ) {}
+
+  async create(createSqueezeDto: SqueezeRequestDto) {
```

Comment:
you can make this more descriptive like saying createSqueeze

Reply:
He hasn’t changed this